cmd/core/pfcp_handlers.go

```diff
@@ -86,16 +91,20 @@ func HandlePfcpAssociationSetupRequest(conn *PfcpConnection, msg message.Message
 	conn.NodeAssociations[addr] = remoteNode
 	log.Info().Msgf("Saving new association: %+v", remoteNode)
 
+	featuresOctets := []uint8{}
+	featuresOctets = append(featuresOctets, setBit(0, 4)) //FTUP
```

Comment:
It'd be better to support configurable options. Let's add FTUP (and UEIP) features to config in order to be able to enable/disable these options.